src/Showoff.jl

```diff
@@ -4,6 +4,12 @@ module Showoff
 
 using Dates
 
+if isdefined(Base, :Grisu)
+    const Grisu = Base.Grisu
```

Comment:
```suggestion
    import Base.Grisu
```

would be better because it will import its exported variables as well ? Not sure Showoff.jl uses them though.

Reply:
> because it will import its exported variables as well 

I agree this is better but what do you mean it will import exported variables? That's only when `using`, no?

Reply:
> That's only when `using`, no?

aha, yes, so this change actually doesn't change the meaning of this program, I guess.